src/pypnusershub/routes.py

```diff
@@ -250,6 +250,27 @@ def login():
         msg = json.dumps({"login": False, "msg": repr(e)})
         return Response(msg, status=403)
 
+@routes.route("/public_login", methods=["POST"])
+def public_login():
+    if current_app.config.get("PUBLIC_ACCESS")["ENABLE_PUBLIC_ACCESS"]:
+        user = (
+            models.AppUser.query.filter(models.AppUser.id_role == current_app.config.get("PUBLIC_ACCESS")["PUBLIC_ID"])
```

Comment:
```suggestion
            models.AppUser.query.filter(models.AppUser.username == current_app.config["PUBLIC_ACCESS"]["PUBLIC_USERNAME"])
```
L’utilisation d’ID sont à éviter car propre à une base de données. Une réinstallation rendrait caduque notre fichier de configuration.